_articles/building-the-best-one-click-sign-in.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: article
+title: "Building The Best One Click Sign In"
+abstract: "One click sign in using device authentication offers convenience and security to the user."
+sitemap:
+  lastmod: 2020-05-18
+---
+
+One click sign in allows users to authenticate in just one click.  Delivering this was the reason DID.app was started. This is our story of implementing one click sign in.
+
+## Why do we need one click sign in?
+
+Users have ‘account creation fatigue’. The average person already has 150+ accounts and the problem is still getting worse. There are so many websites that want you to create an account and it's hard to keep track of them all.
+
+Users want to engage without filling in endless forms, choosing yet another password or confirming their email address.  Users want convenience.  Or, to put it another way, users are lazy; I know I am.
```

Comment:
What about something like. 
"We know users crave this experience because of how many people use password managers. However these just paper over the problems with passwords, and they are tricky for everyone to set up"